engine_file_path: /mnt/ml/yolov8/yolov8s.engine
use_system_default_qos: true
viz: false
